src/Nncase.Tests/Core/UnitTestWeakList.cs

```diff
@@ -26,5 +27,64 @@ public void TestWeakList()
         weakList.Add(3);
         weakList.Add(4);
         Assert.Equal(new WeakReference<object>(1).TryGetTarget(out _), weakList.GetWeakReference(0).TryGetTarget(out _));
+
+#if false
```

Comment:
why leave it with false?

Reply:
The class is't the real weaklist,"GC.(null)" dose‘t work.No sure if it will be fixed after.